tests/test_mutualauthreq.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import absolute_import, division, print_function
+
+import multiprocessing as mp
+import os
+
+import falcon
+import falcon.testing as testing
+
+import requests
+
+from twisted.internet import reactor, ssl
+from twisted.web.server import Site
+from twisted.web.wsgi import WSGIResource
+
+from falcon_mutualauth.mutualauth import Authorize, MutualAuthRequest
+
+
+class HelloResource(object):
+    def on_get(self, req, resp):
+        resp.body = 'hello alice!'
+
+
+class TestTLSCerts(testing.TestCase):
+    def setUp(self):
+        super(TestTLSCerts, self).setUp()
+
+        tests_dir = os.path.dirname(__file__)
+        self._certs_dir = os.path.join(
+            tests_dir,
+            'fixtures',
+            'certs'
+        )
+
+        self._skeypath = self._cert_path('server-key.pem')
```

Comment:
Is it possible to define these methods before SetUp so this code is easier to follow?